src/main/ngapp/accio/src/app/editor/editor.component.ts

```diff
@@ -1,15 +1,50 @@
 import { Component, OnInit } from '@angular/core';
+import { ViewChild, ElementRef } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+import { imageUrls } from '../images';
 
 @Component({
   selector: 'app-editor',
   templateUrl: './editor.component.html',
   styleUrls: ['./editor.component.css']
 })
 export class EditorComponent implements OnInit {
+  // Define urls within component
+  url;
 
-  constructor() { }
+  // inject canvas from html
+  @ViewChild('canvas', { static: true })
+  canvas: ElementRef<HTMLCanvasElement>; 
+  private ctx: CanvasRenderingContext2D;
+  image = new Image();
 
-  ngOnInit(): void {
-  }
+  constructor(
+    private route: ActivatedRoute,
+  ) { }
+
+  ngOnInit() {
+    //returns url user clicked in gallery component
+    this.route.paramMap.subscribe(params => {
+      this.url = imageUrls[+params.get('imgUrl')];
 
+     this.ctx = this.canvas.nativeElement.getContext('2d');
+     this.draw();
+    });
+  }  
+  
+  // Draws the image user selects from gallery on Canvas
+  // currently image stretches to fit canvas, will adjust so canvas
```

Comment:
Please use full sentences that start with a capital letter and end with a period. I can't tell if "currently" is continuing the previous sentence or not.